x-pack/plugins/observability_ai_assistant/scripts/evaluation/scenarios/apm/index.spec.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/// <reference types="@kbn/ambient-ftr-types"/>
+
+import expect from '@kbn/expect';
+import moment from 'moment';
+import { apm, timerange, serviceMap } from '@kbn/apm-synthtrace-client';
+import { RuleResponse } from '@kbn/alerting-plugin/common/routes/rule/response/types/v1';
+import { chatClient, kibanaClient, synthtraceEsClients } from '../../services';
+import { MessageRole } from '../../../../common';
+import { apmErrorCountAIAssistant } from '../../alert_templates/templates';
+
+describe('apm', () => {
+  const ruleIds: any[] = [];
+  before(async () => {
+    const responseApmRule = await kibanaClient.callKibana<RuleResponse>(
+      'post',
+      { pathname: '/api/alerting/rule' },
+      apmErrorCountAIAssistant.ruleParams
+    );
+    ruleIds.push(responseApmRule.data.id);
+
+    await synthtraceEsClients.apmSynthtraceEsClient.clean();
+
+    await synthtraceEsClients.apmSynthtraceEsClient.index(
```

Comment:
These can be parallelised (by passing them all to a single `index` call). It's a little faster because it will then not wait for a refresh.